tests/simpleTest/simple_test_client.hpp

```diff
@@ -74,6 +74,8 @@ class SimpleTestClient {
     ICommunication* comm = bftEngine::CommFactory::create(conf);
 
     SimpleClient* client = SimpleClient::createSimpleClient(comm, id, cp.numOfFaulty, cp.numOfSlow);
+    auto aggregator = std::make_shared<concordMetrics::Aggregator>();
```

Comment:
minor: singleliner?

Reply:
I'm using the aggregator at the end of the test.